front/app/components/UI/RHFFeedback/index.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { useEffect, useState } from 'react';
+import { useFormContext } from 'react-hook-form';
+import Error, { findErrorMessage, TFieldName } from 'components/UI/Error';
+import { injectIntl } from 'utils/cl-intl';
+import { InjectedIntlProps } from 'react-intl';
+import {
+  Text,
+  Title,
+  Box,
+  colors,
+  Icon,
+} from '@citizenlab/cl2-component-library';
+import { scrollToElement } from 'utils/scroll';
+import CloseIconButton from '../CloseIconButton';
+import messages from './messages';
+
+type RHFFeedbackProps = {
+  successMessage?: string;
+} & InjectedIntlProps;
+
+const RHFFeedback = ({
+  successMessage,
+  intl: { formatMessage },
+}: RHFFeedbackProps) => {
+  const [successMessageIsVisible, setSuccessMessageIsShown] = useState(true);
```

Comment:
It doesn't really matter but I think it's easier if the variable and the setter have the same name, so `const [successMessageIsVisible, setSuccessMessageIsVisible] = useState(true);`

Reply:
Totally!